lib/octicons_react/README.md

```diff
@@ -62,6 +62,70 @@ export default () => (
 )
 ```
 
+### `aria-labelledby`
+
+You have the option of adding accessibility information to the icon with the
+[`aria-labelledby` attribute][aria-labelledby] via the `aria-labelledby` prop. Using aria-labelledby referencing the id values of the title element provides the accessible name.
```

Comment:
What is the link here intended to refer to? Specifically in: `[`aria-labelledby` attribute][aria-labelledby]`